### PR TITLE
feat: add whitelist to regexes to skip scrubbing

### PR DESCRIPTION
**Pull request by OrrLumigo**

### CODE REVIEW

File: src/utils/payloadStringify.js [hunk 1]
```diff
@@ -15,22 +17,33 @@ const TRUNCATED_TEXT = '...[too long]';
...
     }
   }
   return regexesList.map((x) => new RegExp(x, 'i'));
 };
 
+export const keyToOmitRegexes = () => {
+  return keyToRegexes();
+};
+
+export const WhitelistKeysRegexes = () => {
```
Comment (nirLumigo) [inline src/utils/payloadStringify.js:43]:
```suggestion
export const whitelistKeysRegexes = () => {
```
Don't we have a linter that is suppose to catch this?

File: src/utils/payloadStringify.js [hunk 2]
```diff
@@ -15,22 +17,33 @@ const TRUNCATED_TEXT = '...[too long]';
 
 const isNativeType = (obj) => nativeTypes.includes(typeof obj);
 
-export const keyToOmitRegexes = () => {
-  let regexesList = OMITTING_KEYS_REGEXES;
-  if (process.env[LUMIGO_SECRET_MASKING_REGEX_BACKWARD_COMP]) {
-    const parseResponse = parseJsonFromEnvVar(LUMIGO_SECRET_MASKING_REGEX_BACKWARD_COMP, true);
+const keyToRegexes = (
+  regexesList = OMITTING_KEYS_REGEXES,
+  backwardCompRegex = LUMIGO_SECRET_MASKING_REGEX_BACKWARD_COMP,
```
Comment (nirLumigo) [inline src/utils/payloadStringify.js:22]:
```suggestion
  backwardCompRegexEnvVarName = LUMIGO_SECRET_MASKING_REGEX_BACKWARD_COMP,
```

File: src/utils/payloadStringify.js [hunk 3]
```diff
@@ -15,22 +17,33 @@ const TRUNCATED_TEXT = '...[too long]';
...
+  envDefinedRegexes = LUMIGO_SECRET_MASKING_REGEX
```
Comment (nirLumigo) [inline src/utils/payloadStringify.js:23]:
```suggestion
  regexesEnvVarName = LUMIGO_SECRET_MASKING_REGEX
```

**Review by nirLumigo — APPROVED** [review]

**Comment by efimk-lu** [comment]
@OrrLumigo I want Saar or Dori to CR this as well, in general I think that as a rule of thumb, nodejs is too sensitive to have one CR, we need to have two by default

**Review by doriaviram — APPROVED** [review]

**Pull request merged**